gitea/release.go

```diff
@@ -46,6 +47,33 @@ func (c *Client) GetRelease(user, repo string, id int64) (*Release, error) {
 	return r, err
 }
 
+// ListReleaseAssets gets all the assets of a release in a repository
+func (c *Client) ListReleaseAssets(user, repo string, id int64) (*Release, error) {
```

Comment:
@stefan-lacatus after talking to @bkcsoft we agreed on naming that in golang code we should use name `Attachment` and in json payloads and url's use GitHub name `asset`